### PR TITLE
Edits for README.adoc

### PR DESCRIPTION
**Pull request by yeekangc**
Edits for the guide.

### CODE REVIEW

**Comment by yeekangc** [comment]
@shin19991207 @gkwan-ibm, please ask @dmuelle to take one final look when we have the final version.  Thank you.

File: README.adoc [hunk 1]
```diff
@@ -9,47 +9,47 @@
...
+The https://grpc.io/[gRPC^] Remote Procedure Call is a technology that implements remote procedure call (RPC) style APIs with HTTP/2. Typically, gRPC uses https://developers.google.com/protocol-buffers/docs/reference/overview[protocol buffers^] to define format of data to be transferred and service interfaces to access it, which include service calls and expected messages. For each service defined in a `.proto` file, gRPC uses the definition to generate the skeleton code for users to implement and extend. Protocol buffers use a binary format to send and receive messages that is faster and more lightweight than the JSON that is typically used in RESTful APIs.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
The https://grpc.io/[gRPC^] Remote Procedure Call is a technology that implements remote procedure call (RPC) style APIs with HTTP/2. Typically, gRPC uses https://developers.google.com/protocol-buffers/docs/reference/overview[protocol buffers^] to define the format of data to be transferred and the service interfaces to access it, which include service calls and expected messages. For each service defined in a `.proto` file, gRPC uses the definition to generate the skeleton code for users to implement and extend. Protocol buffers use a binary format to send and receive messages that is faster and more lightweight than the JSON that is typically used in RESTful APIs.
```

File: README.adoc [hunk 2]
```diff
@@ -9,47 +9,47 @@
...
 
-Protocol buffers allow cross-project support through the `.proto` file. As a result, gRPC clients and servers can run and communicate with each other from different environments. For example, a gRPC client running in Java codebase is able to call a gRPC server from other https://grpc.io/docs/languages/[supported languages^]. This feature of protocol buffers allows for easier integration between services.
+Protocol buffers allow cross-project support through the `.proto` file. As a result, gRPC clients and servers can run and communicate with each other from different environments. For example, a gRPC client running on a Java virtual machine is able to call a gRPC server developed in other https://grpc.io/docs/languages/[supported languages^]. This feature of protocol buffers allows for easier integration between services.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Protocol buffers allow cross-project support through the `.proto` file. As a result, gRPC clients and servers can run and communicate with each other from different environments. For example, a gRPC client running on a Java virtual machine can call a gRPC server developed in any other https://grpc.io/docs/languages/[supported language^]. This feature of protocol buffers allows for easier integration between services.
```

File: README.adoc [hunk 3]
```diff
@@ -9,47 +9,47 @@
...
 The `query` service implements four RESTful APIs by using four different gRPC streaming methods.
 
-* Unary RPC: The client sends a single request and recieves a single response.
+* Unary RPC: The client sends a single request and receives a single response.
 * Server streaming RPC: The client sends a single request and the server returns a stream of messages.
 * Client streaming RPC: The client sends a stream of messages and the server responds with a single message.
 * Bidirectional RPC: Both client and server send a stream of messages. The client and server can read and write messages in any order.
 
-image::architecture.png[gRPC application architecture that the system service provides gRPC server and the query service makes different gRPC streaming calls, align="center"]
+image::architecture.png[Application architecture of the gRPC application covered in guide where a system service acts as the gRPC server and a query service acts a client that makes different gRPC streaming calls, align="center"]
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
image::architecture.png[Application architecture of the gRPC application covered in guide, where a system service acts as the gRPC server and a query service acts a client that makes different gRPC streaming calls, align="center"]
```

**Review by dmuelle — COMMENTED** [review]
ID review- a few minor suggestions

File: README.adoc [hunk 4]
```diff
@@ -475,7 +478,7 @@ QueryIT.java
 ----
 include::finish/query/src/test/java/it/io/openliberty/guides/query/QueryIT.java[]
 ----
-Each test case tests one of the methods for instantiating a gRPC client.
+Each test case tests one type of the gRPC calls that have been implemented.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Each test case tests one type of the gRPC calls that you implemented.
```

**Review by dmuelle — APPROVED** [review]
LGTM, thanks

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**